ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install_targets.cmake

```diff
@@ -73,6 +73,11 @@ function(ament_cmake_symlink_install_targets)
           "'${target}' is an imported target")
       endif()
       list(APPEND target_files "$<TARGET_FILE:${target}>")
+      get_target_property(target_type "${target}" TYPE)
+      if("${target_type}" STREQUAL "SHARED_LIBRARY")
+        list(APPEND target_files "$<TARGET_LINKER_FILE:${target}>")
+        list(REMOVE_DUPLICATES target_files)
```

Comment:
I don't think this line will do what you would like to achieve (avoid duplicates on non-Windows platforms). Again, the generator expression won't be evaluated at this point in time.

Maybe only conditionally append `TARGET_LINKER_FILE` on Windows?